task/rhacs-image-check/3.71/README.md

```diff
@@ -4,37 +4,89 @@ Check an image against RHACS build and deploy lifecycle policies to validate a p
 - **`output_format`**:  Examples: _**table**, csv, json, junit_
-- **`rox_central_endpoint`**: Secret containing the address:port tuple for StackRox Central. Default: _**rox-central-endpoint**_
-- **`rox_api_token`**: Secret containing the StackRox API token with CI permissions. Default: _**rox-api-token**_
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central. Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_config_dir`**: Path to the roxctl config directory within the `roxctl-config` workspace (if machine to machine authentication is used). Mutually exclusive with **`rox_token_file`**. The path should be prefixed with `/roxctl-config`. Examples: _"/roxctl-config", **""**_.
+- **`rox_token_file`**: Path to the API Token file (if API Token authentication is used). Mutually exclusive with **`rox_config_dir`**. The path should be prefixed with `/rox-api-token-auth`. Examples: _**""**, "/rox-api-token-auth/rox_api_token"_.
+- `rox_image`: The image providing the roxctl tool (optional). Default: quay.io/stackrox-io/roxctl:4.4.2 (this is also the minimum version working with this task). 
+
+One of the **`rox_config_dir`** or **`rox_token_file`** parameter is required for the authentication against the remote Central to work.
+
+## Workspaces
+
+- **roxctl-config**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces) containing the configuration for roxctl. Used to authenticate with the remote central using short-lived tokens. The content of this worksapce is ideally populated by a rhacs-m2m-authenticate TaskRun. This workspace is mutually exclusive with the `rox-api-token-auth` workspace.
+- **rox-api-token-auth**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces) containing a rox token file. Used to authenticate with the remote central. It is **strongly** recommended that this workspace be bound to a Kubernetes `Secret`. This workspace is mutually exclusive with the `roxctl-config` workspace.
 
 ## Usage
 
 Checks images that have been pushed to a registry.  This enables scanning regardless of whether the build is using traditional Docker-based approaches, hosted/SaaS-based approaches where the Docker socket may not be directly available, or rootless approaches like `kaniko` and `buildah`.
 
 If the image violates one or more enforced policies, this task will return a failure and cause the pipeline run to fail.
 
+### Configure using machine to machine authentication
+
+Check the [documentation](https://docs.openshift.com/acs/operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access_configure-short-lived-access) to configure the trust with the OIDC token issuer.
```

Comment:
Same as above, I think we should document thins once in the m2m task and refer to it here instead of duplicating it across tasks